.circleci/config.yml

```diff
@@ -240,6 +240,27 @@ jobs:
             just prepare-json
             just simulate-council # simulate again to make sure the json is still valid
 
+  just_simulate_single:
```

Comment:
Isn't this identical to `simulate`?

Reply:
I am not sure, I don't understand why the `simulate` was not working let me retry again.

Reply:
I just realized that @blmalone introduce the `just_simulate_single_improvements` feature 2 days ago. 
so I have update the code the simulation with this one. 

However, I think we can still remove the `simulate` that is currently mention in this discussion:  https://github.com/ethereum-optimism/superchain-ops/blob/341d9922d72d6f05e6e963cf02c6af9dbc53d756/.circleci/config.yml#L28

Reply:
fix in commit 0402aa0